spec/shoryuken/middleware/server/exponential_backoff_retry_spec.rb

```diff
@@ -33,7 +35,9 @@
 
       expect(sqs_msg).not_to receive(:change_visibility)
 
-      expect { subject.call(TestWorker.new, queue, sqs_msg, sqs_msg.body) { raise } }.to raise_error
+      expect {
```

Comment:
@phstc Even though @houndci is complaining, I left it as a multi-line block, since you use curly brackets in other parts as well. I hope that's fine for you.